pkg/daemon/ceph/client/filesystem.go

```diff
@@ -174,6 +177,23 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPoolSize(context *clusterd.Context, clusterName, fsName string, dataPool []*model.Pool, metdataPool *model.Pool) error {
+	// updating the dataPool size
+	metaDataPoolname := fsName + metaDataPoolSuffix
+	err := SetPoolProperty(context, clusterName, metaDataPoolname, "size", strconv.Itoa(int(metdataPool.ReplicatedConfig.Size)))
+	if err != nil {
+		return errors.Wrapf(err, "failed to update metadataPool replica for filesystem %q", fsName)
+	}
+	//updating dataPool size
+	dataPoolname := fsName + dataPoolSuffix
```

Comment:
Instead of regenerating the pool name here and then appending the numeric suffix later, let's factor out the code in `newFS()` that generates the data pool names and then re-use the new helper method. Then we don't need new code to generate the pool names.